From b301fb7bc4201e9d5d94001dbe9ddb031892f7f5 Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Mon, 22 Oct 2012 16:32:52 -0700
Subject: libsuspend: Use autosleep first if available

Change the order to look for sysfs nodes to do autosleep first,
then early suspend.

Change-Id: Iff34b9354dd8e5104befd05e8097962d7f7eee7f
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 libsuspend/autosuspend.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libsuspend/autosuspend.c b/libsuspend/autosuspend.c
index eb1f66e..8cb50d4 100644
--- a/libsuspend/autosuspend.c
+++ b/libsuspend/autosuspend.c
@@ -33,12 +33,12 @@ static int autosuspend_init(void)
         return 0;
     }
 
-    autosuspend_ops = autosuspend_earlysuspend_init();
+    autosuspend_ops = autosuspend_autosleep_init();
     if (autosuspend_ops) {
         goto out;
     }
 
-    autosuspend_ops = autosuspend_autosleep_init();
+    autosuspend_ops = autosuspend_earlysuspend_init();
     if (autosuspend_ops) {
         goto out;
     }
-- 
1.7.4.1

